action.yml

```diff
@@ -134,15 +134,17 @@ runs:
         username: ${{ inputs.registry_username }}
         password: ${{ inputs.registry_token }}
 
+    - run: echo "${{ github.action_path }}" >> "$GITHUB_PATH"
```

Comment:
Documentation would be welcome. and I have missed the content. Why this is needed?

Reply:
I have added some description to the action and to the commit message.